frontend/app/route-helpers/apply-route-helpers.server.ts

```diff
@@ -144,3 +145,18 @@ export function startApplyState({ id, session }: StartArgs) {
 
   return initialState;
 }
+
+export type AgeCategory = 'children' | 'youth' | 'adults' | 'seniors';
+
+export function getAgeCategoryFromDateString(date: string) {
+  const age = getAgeFromDateString(date);
+  return getAgeCategoryFromAge(age);
+}
+
+export function getAgeCategoryFromAge(age: number): AgeCategory {
```

Comment:
I guess youth should be `youths` if you're going to pluralize categories, but that's really very minor and just a nitpick.

Reply:
I took the category names form Stat Can.
https://www.statcan.gc.ca/en/concepts/definitions/age2

Reply:
huh, TIL